internal/dockerfile2dot/build.go

```diff
@@ -90,6 +106,18 @@ func BuildDotFile(simplifiedDockerfile SimplifiedDockerfile, legend bool) string
 			)
 		}
 	}
+	if layers {
+		if len(simplifiedDockerfile.LayersNotStage) > 0 {
+			graph.AddSubGraph("G", "cluster_layers_not_stage", map[string]string{"label": "Layers NOT stage "})
```

Comment:
```suggestion
			graph.AddSubGraph("G", "cluster_layers_not_stage", map[string]string{"label": "Before First Stage"})
```

Reply:
Done in #5ab538313ca64ca482ffefdf9106c46a582887e4